features/step_definitions/secrets.js

```diff
@@ -79,6 +79,8 @@ module.exports = function server() {
         }).then((response) => {
             this.jobId = response.body[0].id;
             this.secondJobId = response.body[1].id;
+            if (2 in response.body) this.thirdJobId = response.body[2].id;
+            this.lastJobId = response.body[response.body.length - 1].id;
```

Comment:
`lastJobId` is used for `the event is done`. The end of the event is synonymous with the end of the last job.